### PR TITLE
feat: add empty lines between methods in classes.

### PR DESCRIPTION
**Pull request by dpurp**
Summary:
Adds an empty line between functions in a class.

### CODE REVIEW

File: src/test/java/com/google/javascript/gents/singleTests/access_modifiers.ts [hunk 1]
```diff
@@ -10,10 +10,15 @@ class A {
   private e: number;
   // Method access
```
Comment (gmoothart) [inline src/test/java/com/google/javascript/gents/singleTests/access_modifiers.ts]: I'm curious if there is an easy way to add a space between fields and the constructor as well - what if you added before rather than after the method body? No need to hold up this PR if it isn't straightforward.

File: src/main/java/com/google/javascript/gents/GentsCodeGenerator.java [hunk 2]
```diff
@@ -54,11 +56,20 @@ protected void add(Node n, Context ctx) {
...
     Node nextNode = n.getNext();
     if (nextNode != null) {
       int lineSpacing = nextNode.getLineno() - n.getLineno();
-      if (lineSpacing > 1) {
+
+      // Add an empty line if:
+      //   - the next node is further away than the next line
+      //   - the next node is a class function (but the current node is not a comment)
+      if (lineSpacing > 1 ||
+          (nextNode.getToken() == Token.MEMBER_FUNCTION_DEF ^ n.getToken() == Token.EMPTY)) {
```
Comment (gmoothart) [inline src/main/java/com/google/javascript/gents/GentsCodeGenerator.java]: One of the newline conditions here is "current node is empty and next token is not a member function" - i.e. add an newline after all empty tokens except those followed by a member function. That seems non-intuitive to me; why is it what we want?

Comment (dpurp) [inline src/main/java/com/google/javascript/gents/GentsCodeGenerator.java]: Otherwise, the output will have an extra new line after the `//inline comment`

See below:

Input:
```javascript
const F = goog.defineClass(null, {
  // inline comment

  /**
   * block comment
   */
  constructor: function() {
  },
});
```

Output without check:
```typescript
class F {
  // inline comment


  /**
   * block comment
   */
  constructor() {}
}
```

File: src/test/java/com/google/javascript/gents/singleTests/access_modifiers.ts (hunk 1 above)
Comment (dpurp) [inline src/test/java/com/google/javascript/gents/singleTests/access_modifiers.ts]: Probably, but I'd rather do it in another PR.

File: src/main/java/com/google/javascript/gents/GentsCodeGenerator.java (hunk 2 above)
Comment (dpurp) [inline src/main/java/com/google/javascript/gents/GentsCodeGenerator.java]: Not sure what I was thinking here, but that does indeed work.  (Updated accordingly)
